src/test/resources/reference.conf

```diff
@@ -47,8 +47,9 @@ nih {
 }
 
 elasticsearch {
-  urls = "dummy"
+  urls = "dummy:123"
```

Comment:
this change was necessary because, with the implementation of discoverGroupNames, the tests now resolve FireCloudConfig.ElasticSearch where previously they didn't, and the value we had for `urls` was invalid and caused an error.